node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -58,14 +61,17 @@ type WaitForMpathResult struct {
 }
 
 var (
-	TimeOutMultipathCmd  = 60 * 1000
-	TimeOutMultipathdCmd = 10 * 1000
-	TimeOutBlockDevCmd   = 10 * 1000
-	TimeOutSgInqCmd      = 3 * 1000
+	TimeOutMultipathCmd                  = 60 * 1000
+	TimeOutMultipathdCmd                 = 10 * 1000
+	TimeOutBlockDevCmd                   = 10 * 1000
+	TimeOutSgInqCmd                      = 3 * 1000
+	MultipathdFilterMpathAndVolumeId     = []string{"%w", "%d"}
```

Comment:
I think these are more "wildcards" than they are a "filter"

Reply:
what do you mean? I don't fully understand you

Reply:
`"%w", "%d"` tells the command which fields ("columns") to show, not which entries ("rows"/"lines"), hence IMO they are not filtering, but are being "expanded"/"resolved" as [wildcards](https://linux.die.net/man/8/multipathd)

Reply:
I like more "format" instead of "filter", but if you think "wildcards" is better,I am ok with that

Reply:
we already have "format" for the [combination](https://github.com/IBM/ibm-block-csi-driver/blob/27a35f2f87b22d11760d98dc6ac1e06c380033d8/node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go#L674-L675) of these ("wildcards") with the separator, so I don't think we should use it here as well, and I don't have any better name, and that is the name used in the official manual page

Reply:
ok I will change any "filter" to "wildcards"

Reply:
done